onecodex/analyses.py

```diff
@@ -246,6 +254,16 @@ def _metadata_fetch(self, metadata_fields, label=None):
 
         return magic_metadata, magic_fields
 
+    @property
+    def all_nan_classification_ids(self):
```

Comment:
Can we make this a "private" property, i.e. `_all_nan_classification_ids`? It's a pretty internal thing that we might not want to commit to having in the public API (yet)

Reply:
Sure, that makes sense (although it will necessitate renaming the attribute on `OneCodexAccessor`s something else besides `_all_nan_classification_ids` to distinguish between the two). But also, how does leaving it as-is commit us to anything?

Reply:
I have renamed it